# Generated by Django 5.0 on 2024-02-08 09:56
import logging

from django.db import migrations, models
from django.core import management
logger = logging.getLogger(__name__)


class Migration(migrations.Migration):

    dependencies = [
        ('geonames', '0003_geonamesadm3_it_codice_istat_alter_country_id_and_more'),
    ]

    def fixture_country_istat(apps, schema_editor):
        try:
            management.call_command('country_it_codice_catastale')
            management.call_command('synchgeonamescountries_istat')
        except Exception as ex:
            logger.error("fixture_country_istat: %s" % str(ex))

    operations = [
        migrations.AddField(
            model_name='country',
            name='it_codice_istat',
            field=models.CharField(blank=True, max_length=4, null=True),
        ),
        migrations.RunPython(fixture_country_istat)
    ]
